web/src/views/DiscussionsView.vue

```diff
@@ -117,8 +117,9 @@
         <!-- CHAT AREA -->
         <div class="container flex flex-col flex-grow pt-4 pb-10">
             <!-- REMOVED @click="scrollToElement($event.target)" -->
+            <!-- Removed reference to copyToClipBoard() ; function was moved to Message.vue -->
             <Message v-for="(msg, index) in discussionArr" :key="index" :message="msg" :id="'msg-' + msg.id" ref="messages"
-                @copy="copyToClipBoard" @delete="deleteMessage" @rankUp="rankUpMessage" @rankDown="rankDownMessage"
+                @delete="deleteMessage" @rankUp="rankUpMessage" @rankDown="rankDownMessage"
```

Comment:
Can you explain why did you remove the copy to clipboard from this section?
I don't see any additional code to message.vue that replaces this.

Reply:
Its all right the copytoclipboard method is in Message.vue. the one in DiscussionsView.vue was a test method tgat di nothing, leftovers from debugging